README.md

```diff
@@ -125,6 +124,9 @@ export function unmount(props) {
 }
 ```
 
+## Project roadmap
```

Comment:
Just random change to maybe help drive community involvement in 3.7 or the website/video